src/fstbin/make-grammar-fst.cc

```diff
@@ -80,17 +120,21 @@ int main(int argc, char *argv[]) {
 
     int32 nonterm_phones_offset = -1;
     bool write_as_grammar = true;
+    bool make_mutable = false;
 
     po.Register("nonterm-phones-offset", &nonterm_phones_offset,
                 "Integer id of #nonterm_bos in phones.txt");
     po.Register("write-as-grammar", &write_as_grammar, "If true, "
                 "write as GrammarFst object; if false, convert to "
                 "ConstFst<StdArc> (readable by standard decoders) "
                 "and write that.");
+    po.Register("make-mutable", &make_mutable, "If true, "
```

Comment:
Is there any reason why this binary needs to support this?  In terms of use-cases, for instance?
If it's not likely going to be used I'd rather just keep the changes to the underlying code so people can
use it as they wish.

Reply:
I would argue that this change is useful/necessary. It lets you create both types of GrammarFstTpls. Does it help that if you exclude the argument, the program will make a `ConstGrammarFst`, which should let this be change be obscured from scripts that use `make-grammar-fst`